### ISSUE TITLE
Crash on master

### ISSUE THREAD

**Issue by walles** [walles/moar#64]
```
Please post the following report at <https://github.com/walles/moar/issues>,
or e-mail it to johan.walles@gmail.com.

Version: v1.7.1-15-g962235d
LANG   : sv_SE.UTF-8
TERM   : xterm-256color

GOOS    : darwin
GOARCH  : amd64
Compiler: gc
NumCPU  : 16

panic: screen lines rendering failed, first 1-based input line available was 0 [recovered]
	panic: screen lines rendering failed, first 1-based input line available was 0 [recovered]
	panic: screen lines rendering failed, first 1-based input line available was 0

goroutine 1 [running]:
main.main.func1()
	/Users/walles/src/moar/moar.go:129 +0x6e
panic(0x131ad60, 0xc000010020)
	/usr/local/Cellar/go/1.16.4/libexec/src/runtime/panic.go:965 +0x1b9
main.startPaging.func1(0x14ab948, 0xc0000bc600, 0xc000070020)
	/Users/walles/src/moar/moar.go:266 +0x125
panic(0x131ad60, 0xc000010020)
	/usr/local/Cellar/go/1.16.4/libexec/src/runtime/panic.go:965 +0x1b9
github.com/walles/moar/m.(*ScreenLines).renderScreenLines(0xc0000b1bf0, 0x1, 0x3a, 0xc00034a000, 0x61c375c5)
	/Users/walles/src/moar/m/screenLines.go:42 +0x168
github.com/walles/moar/m.(*Pager)._Redraw(0xc000076000, 0x0, 0x0)
	/Users/walles/src/moar/m/pager.go:171 +0x148
github.com/walles/moar/m.(*Pager).StartPaging(0xc000076000, 0x14ab948, 0xc0000bc600)
	/Users/walles/src/moar/m/pager.go:663 +0x639
main.startPaging(0xc0000eb3b0, 0x0)
	/Users/walles/src/moar/moar.go:282 +0x125
main.main()
	/Users/walles/src/moar/moar.go:239 +0x765
```

**Comment by walles** [comment]
Repro: `./moar.sh < /dev/null`

**Comment by walles** [comment]
Fixed by 47036defd4200b6f04b1d9c2dbbe079adafd431b.
